source/reference/minio-server/minio-server.rst

```diff
@@ -171,6 +171,54 @@ The command accepts the following arguments:
    Omit to direct MinIO to generate a dynamic port at server startup. The
    MinIO server outputs the port to the system log.
 
+.. mc-cmd:: --ftp
+   :optional:
+   
+   Enable and configure an File Transfer Protocol (``FTP``) or File Transfer Protocol over SSL/TLS (``FTPS``) server.
+   Use multiple times to specify an address port or a passive port range of addresses as key-value pairs.
+
+   Valid keys:
+
+   - ``address``, which takes a single port to use for the server, typically ``8021``
+   - ``passive-port-range``, which restricts the range of potential ports the system assigns to FTP
+   - ``tls-private-key``, which takes the path to the user's private key for accessing the MinIO deployment by TLS
+   - ``tls-public-cert``, which takes the path to the certificate for accessing the MinIO deployment by TLS
+
+   MinIO uses the server's certificate chain automatically if TLS is already enabled.
+   You only need to specify a certificate and private key to terminate certificates with a different domain for your FTP server.
```

Comment:
I think the first sentence might be confusing.

"Omit ``tls-private-key`` and ``tls-public-key`` to direct MinIO to use the default TLS keys for the MinIO deployment. See :ref:`minio-tls` for more information.

Since MinIO can have many TLS keys, I assume we use the default private.key and public.crt here.

Reply:
Done.